CHANGELOG.md

```diff
@@ -13,7 +13,10 @@
   * `xxx-zookeeper-headless` -> `xxx-zookeeper-nodes`
   * `xxx-connect` -> `xxx-connect-api`
 * Cluster Operator moving to Custom Resources instead of Config Maps
-* TLS support has been added to Kafka and Zookeeper
+* TLS support has been added to Kafka, Zookeeper and Topic Operator. The current channels are now encrypted.
```

Comment:
_The current channels are now encrypted._ seems strange. wouldn't _All channels are encrypted._ be better?

Also the bullet points below are strange. If I didn't know what have we implemented I wouldn't know it from this.

Reply:
In my mind was ...

"The following channels are now encrypted" 

listing what channels I'm referring to, in the list below.

Reply:
That makes sense. But then it should probably something like this, or?

The following channels are encrypted:
* Zookeeper cluster communication
* Kafka cluster commbunication
* Communication between Kafka and Zookeeper
* Communication between Topic Operator and Kafka / Zookeeper

Reply:
Yeah that's fine with me. I'll change it.